package.json

```diff
@@ -25,7 +25,6 @@
     "build": "lerna run build",
     "prepare": "bun run build",
     "lerna": "lerna",
-    "test": "bun run check",
```

Comment:
`test` running `check` is misleading API, we should add `test` once our packages define `test` scripts, then run `bun run lerna run check --since <ref> && bun run lerna run test --since <ref>` as a code quality check.

Reply:
Also, in the case that tests/checks are run incrementally, we need to pass a `--since` argument to `lerna run` by invoking `bun run lerna run check` instead of `bun run check`. So, either all packages need to define a script that runs both tests and checks, or we run both test and check separately at the top level.